README.md

```diff
@@ -7,6 +7,12 @@ This is the RAS Collection Instrument micro-service, responsible for the uploadi
 
 The API is specified [here](./API.md).
 
+## Environment
+This requires pipenv and tox to be installed:
+```
+pip install pipenv
+pip install tox
```

Comment:
I think tox is only required for testing so should maybe be in that section?

Reply:
Good call, changing